### PR TITLE
DEV-3202-new-go-exercicse-be-zero

### PR DESCRIPTION
**Pull request by Hamzaelkhatri**

### CODE REVIEW

**Review by tiagocollot — APPROVED** [review]
Everything seems to be correct!

File: subjects/bezero/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+## be-zero
```
Comment (tiagocollot) [inline subjects/bezero/README.md:1]: The title should be "bezero"

**Review by tiagocollot — CHANGES_REQUESTED** [review]

File: subjects/bezero/README.md (hunk 1 above)
Comment (Hamzaelkhatri) [reply subjects/bezero/README.md:1]: I guess not necessary. 
I just separate two word with '-' like kebab-case.
The title could be Bezero , bezero or be-zero

Comment (Hamzaelkhatri) [reply subjects/bezero/README.md]: Fixed

**Review by tiagocollot — APPROVED** [review]

**Pull request closed without merging**